install_config/install/deploy_router.adoc

```diff
@@ -350,12 +350,12 @@ named `www.example.test`, see the example shown below:
       -signkey example-test.key -out example-test.crt
 ----
 
-Generate a route configuration file using the above certificate and key.
-Make sure to replace servicename `my-service` with the name of your service.
+Generate a route using the above certificate and key.
 
 ----
-# servicename="my-service"
-# echo "
+$ oc create route edge secured-edge-route --service=my-service --hostname=www.example.test --key=example-test.key --cert=example-test.crt
```

Comment:
Better to split this into multiple lines, all but the last ending in backslash.  Because the rendering of the docs on the portal is (still!) very vertical, i suggest line break after `secured-edge-route` and also after every option.  For more readability, you can omit the `=` between option name and option value, but that particular change is ... optional. :smile: